lib/services/model/request/electrum_request_types.dart

```diff
@@ -31,7 +31,7 @@ class _VersionReq extends _ElectrumRequest {
   @override
   List<dynamic> get params => [
         "electrs/0.10.5",
-        ["1.1", "1.4"]
+        ["coconut-wallet", "1.4"]
```

Comment:
파라미터가 2차원 배열인데요. 1차원 배열 0번째 요소가 클라이언트 이름이고 1번째 요소는 버전 정보인데요. 이게 배열일 수 있고 단일 문자열 값일 수 있어요. 제가 처음에 코딩해둔게 잘못 적혀있었는데 배열 0번째 요소는 프로토콜 미니멈 버전이에요. 근데 저희가 블록헤더 조회하는 함수를 쓰는데 1.4버전에서 지원해서 1.4만 호출해야될 것 같아요

```suggestion
["coconut-wallet", "1.4"]
```

https://electrum-protocol.readthedocs.io/en/latest/protocol-methods.html#server-version